site/app/templates/admin/Extensions.twig

```diff
@@ -43,12 +43,24 @@
                 Or Pick a New Due Date:
                 <input name="late_calendar" id="late-calendar" class="option-input" type="text" autocomplete="off" onchange="setLateDays()">
             </label>
+            <label for="reason-for-exception">
+                Reason for Extension:
+                <div class="drop-down" id="config-drop-down">
+                    <select name="reason_for_exception" id="reason-for-exception">
+                        <option>{{ "unspecified" }}</option>
+                        {% for reason in reasons %}
+                            <option value="{{ reason }}">{{ reason }}</option>
+                        {% endfor %}
+                    </select>
+                </div>
+            </label>
             <a class="btn btn-primary" href="javascript:updateHomeworkExtension()">Submit</a>
         </fieldset>
         <fieldset>
             <legend class="option-title" id="csv-upload-title">Multiple Student Entry Via CSV Upload</legend>
             <label for="csv-upload" id="csv-upload-format">
-                Do not use column headers. CSV must be of the following form: student_id, gradeable_id, days_of_extension
+                Do not use column headers. CSV must be of the following form: student_id, gradeable_id, days_of_extension, reason_for_extension<br>
```

Comment:
Add a period:
`Do not use column headers. CSV must be of the following form: student_id, gradeable_id, days_of_extension, reason_for_extension.<br>`